e2e/testdata/app-inspect.golden

```diff
@@ -17,17 +17,17 @@
     "Services": [
         {
             "Name": "api",
-            "Image": "python:3.6",
+            "Image": "docker.io/library/python:3.6@sha256:52f872eae9755743c9494e0e3cf02a47d34b42032cab1e5ab777b30c3665d5f1",
```

Comment:
Hope those sha256 will be always available? Maybe the tests are now too fragile as they depend on hub?